### ISSUE TITLE
[1.12.2] Luggage dosent follow to other dimensions

### ISSUE THREAD

**Issue by Morpheus1101** [OpenMods/OpenBlocks#861]
Luggage dosent follow through to other dimensions and upon return to overworld, the luggage is lost

**Comment by boq** [comment]
"Not following" - it has always done that, since it uses fairly unmodified dog/cat AI. So this is more or less feature requests.

"Lost" - can't reproduce. What could have happened is that it might've hit portal while trying to get to you and teleported to nether. Can you re-check?

**Comment by Morpheus1101** [comment]
Yeah i thought about the portal thing an went back, but it was still very much MIA

**Comment by boq** [comment]
Have you tried moving away from portal? It should teleport (as I said, dog/cat AI).
